### PR TITLE
Add profile Maheshpatro.json

### PR DESCRIPTION
**Pull request by Maheshpatro**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1510"></a>

### CODE REVIEW

File: public/data/Maheshpatro.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Mahesh Kumar Patro",
+  "type": "personal",
+  "bio": "Open Source Enthusiast | Devops Learner ",
```
Comment (schmelto) [inline public/data/Maheshpatro.json:4]:
```suggestion
  "bio": "Open Source Enthusiast | Devops Learner",
```

Comment (Maheshpatro) [reply public/data/Maheshpatro.json]: Thank you for update :)

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good to me! 👍

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Pull request merged**